README.md

```diff
@@ -1 +1,272 @@
-# Porag-s-Awesome-Leaderboard
\ No newline at end of file
+# Porag's-Awesome-Leaderboard
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./porag-logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h3><b>Porag's-Awesome-Leaderboard</b></h3>
+  <h5>The Awesome-Leaderboard web project displays scores submitted by different players. It also allows you to submit your score. All data is preserved to the external Leaderboard API service.<h5>
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 Porag's Awesome Leaderboard<a name="about-project"></a>
+
+This project **Awesome Leaderboard** is a project that demonstrates the use of **HTML & CSS and Javascript Moduler development**.
+
+With External **API consumption**, it also demonstrates the proper use of **Async and Await** to send and receive **data**.
+
+Built using **WEBPACK** following **GitFlow** it is  very simple yet powerful project served by **WEBPACK DEV SERVER** . Along with dynamic modification of the DOM with Events including all the coding best practices. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JS</li>
+  </ul>
+
+   <ul>
+    <li>webpack</li>
+  </ul>
+</details>
+  
+  <!-- - **WEBPACK BASICS HIGHLIGHTS**
+   <img src="./WebpackBasicByPorag.png" alt="webpack basics config highlights" width="auto"  height="auto" /> -->
+ 
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Local webpack dev server</li>
+    <li>Optionally can be Deployed Using GitHub Pages</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Use of medium-fidelity wireframe attached below to create a UI.**
+
+   <img src="./leaderboard_wireframe.png" alt="Leaderboard wireframe" width="auto"  height="auto" />
+
+- **Use of callbacks and promises.**
+- **Use of async and await.**
+- **Use proper ES6 syntax and ES6 modules to write modular JavaScript.**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Please follow the link for a live demo.
+
+- [Porag's Awesome Leaderboard]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Your link is broken 😨 Kindly fix this link to make your readme descriptive and professional 😊